BoxBrowseSDK/BoxBrowseSDK/Categories/NSBundle+BOXBrowseSDKAdditions.m

```diff
@@ -13,14 +13,9 @@ @implementation NSBundle (BOXBrowseSDKAdditions)
 
 + (NSBundle *)boxBrowseSDKResourcesBundle
 {
-    static NSBundle *browseSDKResourceBundle = nil;
-    static dispatch_once_t predicate;
-    dispatch_once(&predicate, ^{
-        NSBundle *browseSDKBundle = [NSBundle bundleForClass:[BOXBrowseSDKResourceLocator class]];
-        NSString *browseSDKResourceBundlePath = [browseSDKBundle.bundlePath stringByAppendingPathComponent:@"BoxBrowseSDKResources.bundle"];
-        browseSDKResourceBundle = [NSBundle bundleWithPath:browseSDKResourceBundlePath];
-    });
-
+    NSBundle *browseSDKBundle = [NSBundle bundleForClass:[BOXBrowseSDKResourceLocator class]];
```

Comment:
Device is not as performant as simulator, and while scrolling we could be instantiating hundreds of these. Can we just quickly test to see if this might affect scrolling behavior? I am not yet convinced we want to skip all optimization for this.